js/apps/admin-ui/maven-resources/theme/keycloak.v2/admin/messages/messages_en.properties

```
@@ -3476,4 +3476,8 @@ signatureMaxExp=Max expiration
 signatureMaxExpHelp=Maximum expiration allowed for the JWT. Tokens need to be generated right before authentication. After this period will be considered invalid because they are too old. If undefined the default value is 60 seconds.
 fileNameDialogTitle=Save as
 fileName=File name
-syncUsersStarted=User sync has started.
\ No newline at end of file
+syncUsersStarted=User sync has started.
+openIdVerifiableCredentials=OpenID for Verifiable Credentials
+openIdVerifiableCredentialsHelp=This section is used to configure settings related to OpenID for Verifiable Credential Issuance (OID4VCI).
+oid4vciEnabled=OID4VCI Enabled
```

I tend to prefer this alternative wording. Could you check consistency with other configurations?
```suggestion
oid4vciEnabled=Enable OID4VCI
```